### PR TITLE
[wip] Adding Minio option for object storage when not running in a cloud provider

### PR DESCRIPTION
**Pull request by pburgisser**

### CODE REVIEW

File: argocd/minio/values.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  secretKey: FqYB6mV5YtwzVs9JNTNjsSEmmHY7XsXZ
+  metrics:
+    serviceMonitor:
+      enabled: true
+  buckets:
+    - name: thanos
+      policy: none
+      purge: false
+  persistence:
+    enabled: false
```
Comment (mcanevet) [inline argocd/minio/values.yaml:12]: How is you editor configured to not add new line at end of file?

Comment (pburgisser) [reply argocd/minio/values.yaml:12]: @mcanevet 
C'est le config de Visual Studio de base. Par contre le pipeline ne passe pas, il abandonne en cours de route (timeout) visiblement le make debug retourne pas mal de valeur à vide

File: argocd/apps/values.yaml [hunk 2]
```diff
@@ -29,3 +29,5 @@ apps:
     enabled: true
   vault:
     enabled: true
+  minio:
+    enabled: true
```
Comment (mcanevet) [inline argocd/apps/values.yaml:33]: Could you please set this to `false` as default and set it to `true` in `k3s-docker` module?

File: argocd/minio/values.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+minio:
+  accessKey: ZQQhDzrKcSB2eYntvQEcfE5m8xgBwdMd
```
Comment (mcanevet) [inline argocd/minio/values.yaml:2]: Can you please try to create a secret in Vault using Terraform and use the `existingSecret` field instead? There is an example in the demo-app

**Review by mcanevet — CHANGES_REQUESTED** [review]

**Pull request closed without merging**